experiments/compositions/composition/config/expanders/jinja2-v0.0.1.yaml

```diff
@@ -101,7 +101,7 @@ metadata:
     app.kubernetes.io/part-of: composition
     app.kubernetes.io/instance: jinja2-v0.0.1
     app.kubernetes.io/component: expanders
-  name: jinja2-v0-0-1
+  name: gjinja2-v0-0-1
```

Comment:
qq: does this mean that our current jinja2 expander is based only on grpc so it is called gjinja2?

Reply:
we need to differentiate b/w job based and grpc based by name/type. 
Will send a PR in future where this will become `jinja` and job based one would be renamed to something else like `pjinja`